ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Block/Cardano.hs

```diff
@@ -380,30 +380,10 @@ mkCardanoProtocolInfo genesisByron signatureThreshold transitionConfig initialNo
             shelleyBasedInitialNonce      = initialNonce
           , shelleyBasedLeaderCredentials = []
           }
-        ProtocolParamsShelley {
-            -- Note that this is /not/ the Shelley protocol version, see
-            -- https://github.com/IntersectMBO/cardano-node/blob/daeae61a005776ee7b7514ce47de3933074234a8/cardano-node/src/Cardano/Node/Protocol/Cardano.hs#L167-L170
-            -- and the succeeding comments.
-            shelleyProtVer = ProtVer (SL.natVersion @3) 0
-          }
-        ProtocolParamsAllegra {
-            allegraProtVer = ProtVer (SL.natVersion @4) 0
-          }
-        ProtocolParamsMary {
-            maryProtVer = ProtVer (SL.natVersion @5) 0
-          }
-        ProtocolParamsAlonzo {
-            alonzoProtVer = ProtVer (SL.natVersion @7) 0
-          }
-        ProtocolParamsBabbage {
-            babbageProtVer = ProtVer (SL.natVersion @9) 0
-          }
-        ProtocolParamsConway {
-            conwayProtVer = ProtVer (SL.natVersion @9) 0
-          }
         triggers
         transitionConfig
         emptyCheckpointsMap
+        (ProtVer (SL.natVersion @9) 0)
```

Comment:
We could also use [`eraProtVerHigh`](https://cardano-ledger.cardano.intersectmbo.org/cardano-ledger-api/Cardano-Ledger-Api-Era.html#v:eraProtVerHigh) + [`LatestKnownEra`](https://cardano-ledger.cardano.intersectmbo.org/cardano-ledger-api/Cardano-Ledger-Api-Era.html#t:LatestKnownEra) here.